vnext/Playground/PlaygroundCpp/MainPage.cpp

```diff
@@ -0,0 +1,49 @@
+﻿#include "pch.h"
+#include "MainPage.h"
+#include "MainPage.g.cpp"
+#include "winrt\react.uwp.h"
+#include "SampleModule.h"
+
+using namespace winrt;
+using namespace react::uwp;
+using namespace Windows::UI;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Media;
+
+namespace winrt::PlaygroundCpp::implementation
+{
+    MainPage::MainPage()
+    {
+        InitializeComponent();
+
+		LoadReact();
+    }
+
+	void MainPage::LoadReact()
+	{
+		InstanceSettings settings;
+		settings.UseWebDebugger = true;
+		settings.UseLiveReload = true;
+
+		auto instance = Instance::Create(JSFILENAME);
+
+		// instantiate sample module for registering callbacks for live reload, JS error handling etc.
+		auto module = winrt::make<SampleModule>();
+		instance.RegisterModule(module.as<IModule>());

```

Comment:
I didn't do the testing, but I guess
instance.RegisterModule(*module) will work